test/integration/setup.js

```diff
@@ -138,23 +174,40 @@ const bundleTapeStandalone = async (destDir) => {
 
   const stream = b.bundle();
   const onceStreamEnd = awaitStreamEnd(stream);
-  stream.pipe(fs.createWriteStream(bundleFileName));
+  const destFileStream = fs.createWriteStream(bundleFileName);
+  const onceWritten = new Promise(resolve => {
+    destFileStream.on("close", resolve);
```

Comment:
If you're doing this anyway, consider a `destFileStream.on("error", reject);` too (in case somehow the destination file is non-writable, e.g. due to a permission issue).